Data/ByteString/Builder.hs

```diff
@@ -407,20 +409,6 @@ floatBE = P.primFixed P.floatBE
 doubleBE :: Double -> Builder
 doubleBE = P.primFixed P.doubleBE
 
-------------------------------------------------------------------------------
--- ASCII encoding
-------------------------------------------------------------------------------
-
--- | Char7 encode a 'Char'.
-{-# INLINE char7 #-}
-char7 :: Char -> Builder
-char7 = P.primFixed P.char7
-
--- | Char7 encode a 'String'.
-{-# INLINE string7 #-}
-string7 :: String -> Builder
-string7 = P.primMapListFixed P.char7
-
```

Comment:
Please revert this change, and instead copy these definitions into `RealFloat` (after importing `Prim` as `P`). They're cheap enough to duplicate (inlined anyway!).  And moving them has possible side-effects best avoided.

Reply:
Sure. https://github.com/haskell/bytestring/pull/222/commits/7b6bc511daf39fa5b80efcd9ab516d0cbc8ddae0